Clover/app/build.gradle

```diff
@@ -112,6 +112,10 @@ android {
     sourceSets {
         beta.java.srcDirs = ['src/release/java']
     }
+
+    packagingOptions {
+        pickFirst 'META-INF/LICENSE'
```

Comment:
One of my dependencies in my wrapper library (xml.jackson) was giving me a duplicate file error, and this was a work around

Seemed like a pretty common issue on stackoverflow, but want to look at why some more

Reply:
Yes I've done this before, this is ok.